nilearn/plotting/displays.py

```diff
@@ -946,6 +1021,73 @@ def draw_cross(self, cut_coords=None, **kwargs):
         # the cuts since we are taking the max along one axis
         pass
 
+    def add_graph(self, adjacency_matrix, node_coords,
+                  node_color=None, node_size=None,
+                  edge_cmap=None,
+                  edge_kwargs=None, node_kwargs=None):
+        """Plot undirected graph on each of the axes
+
+            Parameters
+            ----------
+            adjacency_matrix: numpy array of shape (n, n)
+                represents the edges strengths of the graph. Assumed to be
+                a symmetric matrix.
+            node_coords: numpy array of shape (n, 3)
+                3d coordinates of the graph nodes in world space.
+            node_color: color or sequence of colors
+                color(s) of the nodes.
+            node_size: scalar or array_like
+                size(s) of the nodes in points^2.
+            edge_cmap: colormap
+                colormap used for representing the strength of the edges.
+            edge_kwargs: dict
+                will be passed as kwargs to the plt.plot call that plots each
+                edge one by one.
+            node_kwargs: dict
+                will be passed as kwargs to the plt.scatter call that plots all
+                the nodes in one go.
+
+        """
+        if edge_kwargs is None:
+            edge_kwargs = {}
+        if node_kwargs is None:
+            node_kwargs = {}
+
+        if 's' in node_kwargs:
+            raise ValueError("Please use 'node_size' and not 'node_kwargs' "
+                             "to specify node sizes")
+        if 'c' in node_kwargs:
+            raise ValueError("Please use 'node_color' and not 'node_kwargs' "
+                             "to specify node colors")
+
+        if node_size is None:
```

Comment:
Any reason that you are not using "50" as a default in the function signature? I think that it would make it easier for the user to understand this parameter.